src/programs/unblur/unblur.cpp

```diff
@@ -134,8 +135,9 @@ void UnBlurApp::DoInteractiveUserInput()
 		if (FilenameExtensionMatches(input_filename,"eer"))
```

Comment:
is it always eer? or can it be EER?  Perahps FilenameExtensionMatches should just be changed to not be case sensitive?

Reply:
`FilenameExtensionMatches` is already case insensitive.

Reply:
Oh - good :)